src/main/resources/database/1/patch_16.sql

```diff
@@ -61,3 +61,13 @@ FROM (SELECT log.guild_id,
                         FROM repbot_schema.cleanup_schedule clean
                         WHERE clean.guild_id = log.guild_id
                           AND clean.user_id = log.user_id))) rank;
+
+CREATE TABLE repbot_schema.announcements
```

Comment:
I don't think you want to change a migration which was already deployed 🤔

Reply:
Oh yeah. Changed the database version, but didnt moved it to a new script after I noticed this...